Services/DeviceClient.cs

```diff
@@ -105,33 +108,10 @@ public class DeviceClient : IDeviceClient
             await this.SendRawMessageAsync(eventMessage);
         }
 
-        public Task UpdateTwinAsync(Device device)
+        public async Task UpdatePropertiesAsync(Device device)
         {
-            /* TEMPORARY DISABLED
-            if (!this.connected) await this.ConnectAsync();
-
-            var azureTwin = await this.rateLimiting.LimitTwinReadsAsync(
-                () => this.client.GetTwinAsync());
-
-            // Remove properties
-            var props = azureTwin.Properties.Reported.GetEnumerator();
-            while (props.MoveNext())
-            {
-                var current = (KeyValuePair<string, object>) props.Current;
-
-                if (!device.Twin.ReportedProperties.ContainsKey(current.Key))
-                {
-                    this.log.Debug("Removing key", () => new { current.Key });
-                    azureTwin.Properties.Reported[current.Key] = null;
-                }
-            }
-
-            // Write properties
-            var reportedProperties = DictionaryToTwinCollection(device.Twin.ReportedProperties);
-            await this.rateLimiting.LimitTwinWritesAsync(
-                () => this.client.UpdateReportedPropertiesAsync(reportedProperties));
-            */
-            return Task.CompletedTask;
+            // TODO see https://github.com/Azure/device-simulation-dotnet/tree/send-twin-updates
+            // for future PR
```

Comment:
could you expand the comment, not sure what one will be looking for following that link